src/com/lightdev/app/shtm/SHTMLEditorKitActions.java

```diff
@@ -1974,7 +1972,7 @@ static class PrintAction extends AbstractAction implements SHTMLAction {
         static {
             Method printMethod = null;
             try {
-                printMethod = JTextComponent.class.getMethod("print", new Class[] {});
+                printMethod = JTextComponent.class.getMethod("print");
```

Comment:
Keep parameters because there is more than one method with name "print"

Reply:
done